tests/Aspire.Hosting.Azure.Tests/Snapshots/AzureContainerAppsTests.RoleAssignmentsWithAsExisting#01.verified.bicep

```diff
@@ -1,12 +1,12 @@
 ﻿@description('The location for the resource(s) to be deployed.')
 param location string = resourceGroup().location
 
-param storage_outputs_name string
+param storageName string
```

Comment:
Do we know why this parameter name changed? It shouldn't affect anything materially, but would be nice to know why it is changing.

Reply:
Yes, we do, this is because we set the name of the parameter now which results in a call to https://github.com/dotnet/aspire/blob/3fad2ab5e5cec575cddc5329cd18c3c7e08d5121/src/Aspire.Hosting.Azure/AzureProvisioningResourceExtensions.cs#L171 vs https://github.com/dotnet/aspire/blob/3fad2ab5e5cec575cddc5329cd18c3c7e08d5121/src/Aspire.Hosting.Azure/AzureProvisioningResourceExtensions.cs#L200